paper.md

```diff
@@ -0,0 +1,111 @@
+
+### 8. 
+
+![View metadata for all images in "summary.json" (using https://jsoneditoronline.org/).\label{fig:example8}](https://user-images.githubusercontent.com/26821843/188520906-3061ecce-cb0e-4c76-9b81-303731110380.png)
+
+# Method
+
+## Workflow Overview
+Trash AI is trained on the [TACO dataset](http://tacodataset.org/) using [YOLO 5](pytorch.org). Trash AI stores images in [IndexDB](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API) to keep the data primarily browser side and uses [tensorflow.js](https://www.tensorflow.org/js) to keep analysis browser side too. When images are uploaded to the browser, Trash AI provides the prediction of the model as a graphical output. The raw data from the model and labeled images can be downloaded in a batch download to expedite analyses. Any data uploaded to the platform is automatically saved to an [S3 bucket](https://aws.amazon.com/s3/) which we can use to improve the model over time.
+
+## AI Training
+The AI model was developed starting with the TACO dataset which was available with a complimentary Jupyter Notebook on [Kaggle](https://www.kaggle.com/datasets/kneroma/tacotrashdataset). An example notebook was referenced which used the default `YOLO v5 model` [@Jocher:2020] as the basic model to begin transfer learning. Next, transfer learning was completed using the entire TACO dataset to import the image classes and annotations in the YOLO v5 model.
+
+## Limitations
+From our experience, the accuracy of the model varies depending on the quality of the images and their context/background. Trash is a nuanced classification because the same object in different settings will not be considered trash (e.g. a drink bottle on someone's desk vs in the forest laying on the ground). This and other complexities to trash classification make a general trash AI a challenging (yet worthwhile) long term endeavor. The algorithm is primarily trained on single pieces of trash in the image with the trash laying on the ground and the model will likely excel for that use case currently.
+
+# Availability
+Trash AI is hosted on the web at www.trashai.org. The source code is [available on github](https://github.com/code4sac/trash-ai) with an [MIT license](https://mit-license.org/). The source code can be run offline on any machine that can install [Docker and Docker-compose](www.docker.com). [Documentation](https://github.com/code4sac/trash-ai#ai-for-litter-detection-web-application) is maintained by Code for Sacramento on Github and will be updated with each release. The image datasets shared to the tool are in an S3 Bucket that needs to be reviewed before being shared with others due to security and moderation concerns but can be acquired by [contacting the repo maintaniers](https://github.com/code4sac/trash-ai/graphs/contributors). 
```

Comment:
Might also be worth mentioning that there are some instructions (though not exhaustive) for [setting it up and hosting it via your own AWS account](https://github.com/code4sac/trash-ai/blob/manuscript/docs/git-aws-account-setup.md) if anyone was interested in forking, modifying, and/or white labeling the application.  Our deployment model also [uses github actions](https://github.com/code4sac/trash-ai/blob/manuscript/docs/github-actions.md), so the CI/CD setup is built into the code.  These concerns are non-trivial to solve for any team that is looking maintain a project like this with multiple contributors.

Reply:
added https://github.com/code4sac/trash-ai/commit/4ff318f9c429d80adf701fee7131f2329e93818d